lms/templates/header/brand.html

```diff
@@ -14,7 +14,7 @@
 
 <h1 class="hd logo-header navbar-brand">
   <div class="logo">
-    <a class="navbar-brand" href="${reverse('dashboard')}" itemprop="url">
+    <a class="navbar-brand" href="${marketing_link('ROOT')}" itemprop="url">
```

Comment:
I think we need more logic here. For other openedx instance, they may not have marketing site associated. Please make sure, for them, the edX logo still goes to dashboard.